### DISCUSSION TITLE
Cargo binary dependencies remote cache

### DISCUSSION THREAD

**Discussion by aoudiamoncef** [massalabs/massa#2849] [Ideas]
We have only 3 building targets and a bunch of dependencies, we could improve build performance with caching. It's CPU(now) vs Bandwidth

[Build cache](https://doc.rust-lang.org/cargo/guide/build-cache.html)

[Cargo remote](https://www.chevdor.com/post/2019/12/23/cargo-remote)

[sccache](https://github.com/mozilla/sccache)

**Comment by qdrn** [comment]
Could we use that to improve our CI ? We would probably need to pin the rust version.

Related: https://blog.nixbuild.net/posts/2022-03-16-lightning-fast-ci-with-nixbuild-net.html
